src/codegen/formatted_elz_new/tests.rs

```diff
@@ -0,0 +1,227 @@
+use super::formatted_elz;
+
+#[test]
+fn simple_variable() {
+    let formatted_code = format_code("x:int=1;");
+    assert_eq!(formatted_code, "x: int = 1;\n");
+}
+
+#[test]
+fn simple_function() {
+    let formatted_code = format_code("add(x:int,y:int):int=x+y;");
+    assert_eq!(formatted_code, "add(x: int, y: int): int = x + y;\n");
+}
+
+#[test]
+fn simple_function_block() {
+    let formatted_code = format_code("add(x:int,y:int):int{return x+y;}");
+    assert_eq!(
+        formatted_code,
+        "add(x: int, y: int): int {
+  return x + y;
+}
+"
+    );
+}
+
+#[test]
+fn local_variable_in_function() {
+    let formatted_code = format_code("foo():void{x:int=1;}");
+    assert_eq!(
+        formatted_code,
+        "foo(): void {
+  x: int = 1;
+}
+"
+    );
+}
+
+#[test]
+fn function_call_in_function() {
```

Comment:
I don't think we need this. But can test ”if” in function to ensure nested block would work.
Also can test comment in function.